homeassistant/components/mqtt/climate.py

```diff
@@ -593,14 +645,38 @@ def handle_temperature_low_received(msg: ReceiveMessage) -> None:
         @log_messages(self.hass, self.entity_id)
         def handle_temperature_high_received(msg: ReceiveMessage) -> None:
             """Handle target temperature high coming via MQTT."""
-            handle_temperature_received(
+            handle_climate_attribute_received(
                 msg, CONF_TEMP_HIGH_STATE_TEMPLATE, "_attr_target_temperature_high"
             )
 
         add_subscription(
             topics, CONF_TEMP_HIGH_STATE_TOPIC, handle_temperature_high_received
         )
 
+        @callback
+        @log_messages(self.hass, self.entity_id)
+        def handle_current_humidity_received(msg: ReceiveMessage) -> None:
+            """Handle current humidity coming via MQTT."""
+            handle_climate_attribute_received(
```

Comment:
Just for the record
It is okay to enable to setting the current humidity attribute, even if `ClimateEntityFeature.TARGET_HUMIDITY` is not set.